src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int firstLeg;
+    private int secondLeg;
+
+    public RightTriangle(Color color, int firstLeg, int secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    public double getArea() {
+        return firstLeg * secondLeg / 2.0;
```

Comment:
The method 'getArea' should not be directly implemented in the class. Instead, consider creating an interface for calculating the area and implement it in the RightTriangle class.